info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -203,6 +205,9 @@ public void storeBaseElectionInDatabase_checkDatastoreEntityConstructionFromJson
     assertThat(electionEntity.getList("candidateIds")).isEmpty();
     assertThat(electionEntity.getList("candidateIncumbency")).isEmpty();
     assertThat(electionEntity.getString("state")).isEqualTo(STATE);
+    assertThat(((Timestamp) electionEntity.getValue("lastModified").get()).getSeconds()
```

Comment:
Why not compare the timestamp directly instead of doing .getSeconds()?

Reply:
It was initially because I didn't find a compareTo in its documentation

Reply:
This is fixed now!